app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -182,11 +192,73 @@ impl HeaderGeometry {
 
 
 
+// ==============
+// === Colors ===
+// ==============
+
+/// Colors used in the Component Group View.
+///
+/// This structure, used in both [`ide_component_group::View`] and
+/// [`ide_component_group::wide::View`] can be created from single "main color" input. Each of
+/// these colors will be computed by mixing "main color" with application background - for details,
+/// see [`Colors::from_main_color`].
+#[allow(missing_docs)]
+#[derive(Clone, CloneRef, Debug)]
+pub struct Colors {
+    // Note[ao]: The nodes are initialized during construction. The must be samplers, otherwise the
```

Comment:
"The must be samplers" – probably its a mistake.

Reply:
If this is not a FIXME or TODO you dont need to write `[ao]` - blame will remember that.